Assignment 2 - Menu/src/assignment/pkg2/menu/Assignment2Menu.java

```diff
@@ -0,0 +1,90 @@
+/*
+Dylan Tkach
+Java Programming
+CPT-163-27
+ */
+package assignment.pkg2.menu;
+
+import java.util.Scanner;
+
+public class Assignment2Menu {
+
+    public static void main(String[] args) {
+    
+        Scanner keyboard = new Scanner(System.in);
+        
+        int MenuSelection;
+        
+        int charW = 'W';
+        int charA = 'A';
+        int charR = 'R';
+        int charH = 'H';
+        // A declared.
+        int charM = 'M';
+        // M declared.
+        int charE = 'E';
+        // R declared.
+        
+        int total = charW + charA + charR + charH + charA + charM + charM + charE + charR;
+        
+        
+        System.out.println("|=====================================|");
+        System.out.println("| Please choose one of the following: |");
+        System.out.println("|                                     |");
+        System.out.println("| 1. Word Counter                     |");
+        System.out.println("| 2. Is it heresy?                    |");
+        System.out.println("| 3. List of Primarchs                |");
+        System.out.println("| 4. Frame of Reference               |");
+        System.out.println("| 5. Contact Info                     |");
+        System.out.println("| Enter 0 to exit.                    |");
+        System.out.println("|_____________________________________|");
+        
+        System.out.println("\nEnter your selection: ");
+        MenuSelection = keyboard.nextInt();
+        
+        System.out.println("\n\n");
+        
+        
+        switch (MenuSelection){
+            case 0:
+                System.out.println("Thank you for using this program.");
+                break;
+            case 1:
+                System.out.println("You selected choice 1:\n");
```

Comment:
Instead of adding a line to every case telling the user what option they selected you could add a line after their input that tells them what they selected. This would help remove a couple lines of code.